naucse/models.py

```diff
@@ -82,6 +82,10 @@ def license(self):
         return self.root.licenses[self.info['license']]
 
     @reify
+    def github_link(self):
+        return "https://github.com/pyvec/naucse.python.cz/blob/master/lessons/" + self.lesson.slug + "/" + self.slug + "." + self.style
```

Comment:
It would be nice if the property name did not include `github`: forks of the project can be hosted anywhere.
Something like `page_source_link`? `edit_link`? `vcs_link`?